### PR TITLE
implemented book api

### PR DESCRIPTION
**Pull request by HAIOVYI**

### CODE REVIEW

File: src/main/java/mate/academy/mybookshop/repository/BookRepositoryImpl.java [hunk 1]
```diff
@@ -38,6 +39,15 @@ public BookEntity save(BookEntity book) {
         }
     }
 
+    @Override
+    public BookEntity getById(Long id) {
+        try (Session session = sessionFactory.openSession()) {
+            return session.get(BookEntity.class, id);
+        } catch (Exception e) {
+            throw new EntityNotFoundException("Can't get book by id: " + id, e);
```
Comment (makaroshyna) [inline src/main/java/mate/academy/mybookshop/repository/BookRepositoryImpl.java:47]: EntityNotFoundException is meant for service use only.

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -1,3 +1,8 @@
+spring.datasource.url=jdbc:mysql://localhost:3306/book
+spring.datasource.username=root
+spring.datasource.password=password
```
Comment (makaroshyna) [inline src/main/resources/application.properties:3]: Don't push your credentials on remote

**Review by makaroshyna — CHANGES_REQUESTED** [review]
Good job!

File: src/main/resources/application.properties (hunk 2 above)
Comment (HAIOVYI) [reply src/main/resources/application.properties:3]: ok ) thx

File: src/main/java/mate/academy/mybookshop/repository/BookRepositoryImpl.java (hunk 1 above)
Comment (HAIOVYI) [reply src/main/java/mate/academy/mybookshop/repository/BookRepositoryImpl.java:47]: sorry, i don't get it, why? ) 
for me is one place to try catch  and throw this exception for now
maybe, if that was `findById` method, i will do it with Optional and made `orElseThrow` in Service class 
Please help me out with my opinion. )

File: src/main/java/mate/academy/mybookshop/controller/BookController.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/books")
```
Comment (andrii-hoienko) [inline src/main/java/mate/academy/mybookshop/controller/BookController.java:17]:
```suggestion
@RequestMapping("api/books")
```

**Review by andrii-hoienko — APPROVED** [review]
Good job!

**Pull request merged**